### PR TITLE
UI Enhancements, ACL bug fixes, community contributions.

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Bug fixes for ACL 
* UI enhancement and bug fixes

### Description of the Change

- [ACL] - Fix for the header key name while accessing serviceId.
- [ACL] - Fix for user name case sensitive issue.
- [ACL] - Fix for pagination issue in getList file.
- [ACL] - Included api security to jazz_logs
- [ACL - UI fix] - Updated Service creation toast messages and tabs access based permission.
- [Bug fix] - Filter services list with partial string
- [Bug fix] -  Updated pagination for jazz_usermanagement API service to access users details.
- [UI fix] - Use configured error messages 
- [UI fix] - Display pagination, if more more pages are available
- [UI fix] - Edit service flow and  Updated placeholder text in service overview.
- [UI fix] -Disabled delete service button for core services.
- [UI fix] - Fixed service list sort functionality and CSS fixes while loading data.
- [UI fix] - Updated jazz logo and jazz header fixes(services and admin tab will be visible for admin only)

### CODE REVIEW

File: core/jazz_ui/src/config/toastmessages.ts [hunk 1]
```diff
@@ -187,5 +187,26 @@ export const toastMessage = {
...
+    },
+    "metricsResponse": {
+        "dataNull":"Something went wrong while we were fetching service metrics data!",
+        "error400":"Something went wrong while we were fetching service metrics data!",
+        "error401":"Token expired",
+        "error404":"Something went wrong while we were fetching service metrics data!",
+        "error500":"Something went wrong while we were fetching service metrics data!"
+    },
+    "getDeploymentsResponse": {
+        "dataNull":"Something went wrong while we were fetching service metrics data!",
```
Comment (devsatishm) [inline core/jazz_ui/src/config/toastmessages.ts]: Looks incorrect to me (shows metrics data under deployments section?). Is this expected to be shown when there are errors while calling deployments API?

File: core/jazz_ui/src/config/toastmessages.ts [hunk 2]
```diff
@@ -187,5 +187,26 @@ export const toastMessage = {
...
+        "error400":"Something went wrong while we were fetching service metrics data!",
+        "error401":"Token expired",
+        "error404":"Something went wrong while we were fetching service metrics data!",
+        "error500":"Something went wrong while we were fetching service metrics data!"
+    },
+    "getAssetResponse": {
+        "dataNull":"Something went wrong while we were fetching service metrics data!",
```
Comment (devsatishm) [inline core/jazz_ui/src/config/toastmessages.ts]: Same as above. Looks incorrect to me (shows metrics data under assets section?).

File: core/jazz_ui/src/config/toastmessages.ts (hunk 1 above)
Comment (supritat) [reply core/jazz_ui/src/config/toastmessages.ts]: Updated the message

File: core/jazz_ui/src/config/toastmessages.ts (hunk 2 above)
Comment (supritat) [reply core/jazz_ui/src/config/toastmessages.ts]: Updated the message

**Pull request merged**